.github/ISSUE_TEMPLATE/sugestia-zmian.md

```diff
@@ -3,6 +3,8 @@ name: Sugestia zmian
 about: Propozycja dotycząca zmian w dokumentacji
 title: "[SUGESTIA ZMIAN]"
 ---
+**Tekst zapisany między podwójnymi gwiazdkami nie będzie widoczny po utworzeniu issue**
```

Comment:
Are you sure about that? As far as I know a text between `**` (double asterisk) will be in bold.

Reply:
@arkq Poprawione